services-api/src/main/java/io/scalecube/services/exceptions/UnauthorizedException.java

```diff
@@ -7,4 +7,8 @@
   public UnauthorizedException(int errorCode, String message) {
     super(errorCode, message);
   }
+
+  public UnauthorizedException(String message) {
+    super(ERROR_TYPE, message);
+  }
```

Comment:
There's little mess in http around terminologies   forbidden/unauthorized and authentication/authorization.  Plz take a look here:  https://stackoverflow.com/questions/3297048/403-forbidden-vs-401-unauthorized-http-responses. 
So  term _forbidden_ stands for _authroization error_, and _unauthorized_  is for _authentication error_. Looks like we need  new class --  `ForbiddenException`.

Reply:
Added https://github.com/scalecube/scalecube-services/pull/624/commits/98eb543efb827ec0afa068c24e5b2216696574b4